projects/packages/forms/src/dashboard/components/layout/style.scss

```diff
@@ -21,7 +21,7 @@ body.jetpack_page_jetpack-forms-admin {
 	margin: 0;
 	padding: 0;
 	width: 100%;
-	height: 100%;
+	min-height: calc(100vh - 32px); // 32px is the height of the top admin bar
```

Comment:
This height can change deepening on the width I think on mobile it can be bigger. 
On mobile the admin bar is 46px. I am not sure if this makes any difference in this case just wanted to mention  it.

Reply:
Follow-up: 
- https://github.com/Automattic/jetpack/pull/44470